services/movie.py

```diff
@@ -28,13 +32,18 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Avoid using mutable default arguments like `list` in functions, as this can lead to unexpected behavior. Use `None` as the default value and initialize the list inside the function if needed.